### ISSUE TITLE
ModuleDoc false positive on Elixir 1.20.0-rc.1 - reports missing @moduledoc when  present

### ISSUE THREAD

**Issue by e-fu** [rrrene/credo#1247]
## Environment

  - Elixir: 1.20.0-rc.1 (compiled with Erlang/OTP 28)
  - Erlang/OTP: 28 [erts-16.1.2]
  - Credo: 1.7.15

  ## Description

  The `Credo.Check.Readability.ModuleDoc` check reports a false positive on Elixir
  1.20.0-rc.1. Modules with a `@moduledoc` tag are still flagged as missing one.

  ## Reproduction

  Create a file with a `@moduledoc`:

  ```elixir
  # test_moduledoc.ex
  defmodule TestModule do
    @moduledoc """
    This module has a moduledoc.
    """

    def hello, do: :world
  end

  Run credo:

  $ mix credo test_moduledoc.ex

  Expected: No issues reported (module has @moduledoc)

  Actual:
  [R] → Modules should have a @moduledoc tag.
        test_moduledoc.ex:1:11 #(TestModule)

  Notes

  This appears to be a compatibility issue with Elixir 1.20's RC. Similar issues were fixed
  for Elixir 1.18 (v1.7.11) and 1.19 (v1.7.12-13).

**Comment by darkblueorange** [comment]
same with Erlang/OTP 27 [erts-15.2]

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It should be fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Will make a release soon! 👍

**Comment by e-fu** [comment]
Confirmed working on master.

**Environment:**
- Elixir: 1.19.5 / 1.20.0-rc.1
- Erlang/OTP: 28
- Credo: master (via `{:credo, github: "rrrene/credo"}`)

No more false positives on `@moduledoc`. Thanks for the quick fix! 🙏

**Comment by rrrene** [comment]
@e-fu This is now part of Credo 1.7.16 ... Thx for reporting! 🎉
